cmd/credential-mediator/components/Mediator.vue

```diff
@@ -197,6 +205,12 @@
 </template>
 
 <script>
+    /*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
```

Comment:
should this not specify the year range? (see the date range DB added below)

Reply:
Something like "Portions of this software Copyright 2020, The TrustBloc contributors"

Reply:
@troyronda ^^